docs/platforms/python/migration/raven-to-sentry-sdk.mdx

```diff
@@ -1,10 +1,10 @@
 ---
-title: Migration Guide
-description: "Migrate from the older version of our Python SDK."
-sidebar_order: 8000
+title: Migrating from raven to Sentry SDK
```

Comment:
I would remove the "from" to have a title that fits in the sidebar.

Reply:
Changed this to "Migrate from raven"